tests/tools/archive/test_schema.py

```diff
@@ -23,6 +24,8 @@
 from tests.utils.archives import get_archive_file
 
 
+# TODO: Could these tests work with sqlite backend?
+@pytest.mark.requires_pgsql
```

Comment:
Don't think so because the whole point is to create a database in PSQL and in SQLite and then compare the schemas.